Moderation policy (#556)
space-wizards/docs: src/en/wizden-staff/moderator/new-mod-guide.md

- `!mute <userID> <time> <reason>`: Gives the user the muted role, preventing them from sending messages in all channels.

- `!unmute <userID>`: Removes the muted role from a user.

- `!modnote <userID> <note>` !mnote also works for this command. Issues a note to that persons account, this is only visible to other moderators, this is good for keeping track of special circumstances with users.

- `!listcases <userID>`: Lists all cases for a user, mutes, warns, notes, and bans. Very helpful command and should be used before taking any action on a user.

- `!softban <userID>`: Softbans a user, this bans and immediately unbans them, purging their messages from the last 24 hours.

- `!tempban <member> <duration of tempban> <days of messages to be deleted> <reason>` Temporarily bans a user for a specified duration, deleting a specified number of days of their messages.

# Discourse

As a moderator, you will have extra permission on discourse, but first to get those new permissions, you need to contact a head game admin or a head moderator to add you to the EN community moderator group on discourse. You will also need to contact PJB to get full moderator powers.

> Once you have full moderation powers, you will be able to see some new tabs, such as internal and ban appeal discussions. You should not access data of players information under any circumstances.

Reviewer: ```suggestion
Once you have full moderation powers, you will be able to view additional tabs, including internal and ban appeal discussions. You should not access players' information under any circumstances.
```

You will also be able to moderate posts, such as locking, deleting, and moving posts, you should only do this when necessary, and always try to inform the original poster of why you are taking action on their post.

# Asking for help

> Feel absolutely free to ask any questions in our [General Moderation Channel](https://discord.com/channels/310555209753690112/1027317842804219984), we will always answer your questions as best as we can. all moderators were new once, and we understand that it can be confusing at first. If you are ever unsure about taking action on a user, please ask for help in the general moderation channel before taking action.

Reviewer: ```suggestion
Feel absolutely free to ask any questions in our [General Moderation Channel](https://discord.com/channels/310555209753690112/1027317842804219984); we will always answer your questions to the best of our ability. All moderators were new once, and we understand that it can be confusing at first. If you are ever unsure about taking action on a user, please ask for help in the general moderation channel before taking action.
```

You don't need to add the word ‘absolutely’ at all, but that's up to you.

- `!tempban <member> <duration of tempban> <days of messages to be deleted> <reason>` Temporarily bans a user for a specified duration, deleting a specified number of days of their messages.

- `!reason <caseid> <reason>`: Changes the reason for a case ID.

# Discourse

As a moderator, you will have extra permission on discourse, but first to get those new permissions, you need to contact a head game admin or a head moderator to add you to the EN community moderator group on discourse. You will also need to contact PJB to get full moderator powers.

Once you have full moderation powers, you will be able to view additional tabs, including internal and ban appeal discussions. You should not access players' information under any circumstances.

You will also be able to moderate posts, such as locking, deleting, and moving them. You should only do this when necessary and always try to inform the original poster of why you are taking action on their post.

# Asking for help

Feel absolutely free to ask any questions in our [General Moderation Channel](https://discord.com/channels/310555209753690112/1027317842804219984); we will always answer your questions to the best of our ability.

> All moderators were new once, and we understand that it can be confusing at first. If you are ever unsure about taking action on a user, please ask for help in the general moderation channel before taking action.

Reviewer: ```suggestion
# New moderator Guide

This document will attempt to explain the basics of being a moderator for Space Station 14.

# Getting started
Welcome to the moderation team! As a moderator, you play a crucial role in maintaining a positive and welcoming environment for our community. This guide will help you get started with your new moderation powers and tooling.

There are various commands and things to learn. This guide will cover the basics, but feel free to ask any questions in the [General Moderation Channel](https://discord.com/channels/310555209753690112/1027317842804219984) if you need any help!

Every moderator was new once, and we understand that it can be confusing at first.

# Commands

Moderators have access to a variety of commands to help them moderate the server, including:
-`!ban <userID> <reason>`: Bans a user from the server, will also send a message to them via DM from the bot, unless they do not accept DMs from non-friends.

- `!unban <userID>`: Unbans a user from the server. This command should only be used if the person has an accepted ban appeal on the forums.

- `!warn <userID> <reason> <points>`: Issues a warning to a user. This will send them a DM from the bot with the reason for the warning. If a user receives five warnings, they are kicked from Discord. Upon receiving 10 warnings, the user is banned from Discord.

- `!kick <userID> <reason>`: Kicks a user from the server; the user may rejoin at any time.

- `!mute <userID> <time> <reason>`: Gives the user the muted role, preventing them from sending messages in all channels.

- `!unmute <userID>`: Removes the muted role from a user.

- `!modnote <userID> <note>` !mnote also works for this command. Issue a note to that person's account. This is only visible to other moderators, and it is good for keeping track of special circumstances with users.

- `!listcases <userID>`: Lists all cases for a user, mutes, warns, notes, and bans. This is a very helpful command and should be used before taking any action on a user.

- `!softban <userID>`: Softbans a user, this bans and immediately unbans them, purging their messages from the last 24 hours.

- `!tempban <member> <duration of tempban> <days of messages to be deleted> <reason>` Temporarily bans a user for a specified duration, deleting a specified number of days of their messages.

- `!reason <caseid> <reason>`: Changes the reason for a case ID.

# Discourse

As a moderator, you will have extra permissions on Discourse. First, to get those new permissions, contact a head game admin or a head moderator to add you to the EN community moderator group on Discourse. You will also need to contact PJB to get full moderator powers.

Once you have full moderation powers, you will be able to view additional tabs, including internal and ban appeal discussions. You should not access players' information under any circumstances.

You will also be able to moderate posts, such as locking, deleting, and moving them. You should only do this when necessary and always try to inform the original poster of why you are taking action on their post.

# Asking for help

Feel absolutely free to ask any questions in our [General Moderation Channel](https://discord.com/channels/310555209753690112/1027317842804219984); we will always answer your questions to the best of our ability.

All moderators were new at some point, and we understand that it can be confusing at first. If you are ever unsure about taking action on a user, please ask for help in the general moderation channel before taking action.
```
